modules/light-clients/08-wasm/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/cosmos/ibc-go/modules/light-clients/08-wasm
 
-go 1.19
+go 1.21
+
+toolchain go1.21.1
```

Comment:
we don't use it afaik, can drop the toolchain directive (I think similar discussion was made for go.mod in main module)

Reply:
I think that if we run go mod tidy, it comes right back.

I can try removing it though